### DISCUSSION TITLE
Can jQuery slim be used? (it appears to work)

### DISCUSSION THREAD

**Discussion by ekerstein** [IonDen/ion.rangeSlider#790] [Q&A] [answered]
jQuery slim is 25.9kB while jQuery is 32.2kB.

jQuery slim appears to work with ion rangeSlider. I can't seem to find any issues.

Is it not recommended to use jQuery slim with rangeSlider, or is it okay to do so?

**Comment by IonDen** [comment, marked as answer]
Hey, actually it should be fine. Ion.RangeSlider is not jQuery heavy - feel free to use it with Slim

**Comment by ekerstein** [comment]
Thank you so much!
